specification/resources/resource-manager/readme.md

```diff
@@ -150,6 +150,14 @@ input-file:
 - Microsoft.Authorization/preview/2015-10-01-preview/policy.json
 ```
 
+### Tag: package-resources-2018-05
+These settings apply only when `--tag=package-resources-2018-05` is specified on the command line.
+
+``` yaml $(tag) == 'package-resources-2018-05'
+input-file:
+- Microsoft.Resources/stable/2018-05-01/resources.json

```

Comment:
@Tiano2017 can you get changes from upstream.
Now CSharp specific config file has been added
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/readme.csharp.md

Reply:
done. @shahabhijeet can you take one more look?